packages/next/src/build/analysis/get-page-static-info.ts

```diff
@@ -229,6 +233,18 @@ function checkExports(
               )
           }
         }
+
+        if (node.type === 'ExpressionStatement') {
```

Comment:
This doesn’t guarantee that there’s no other non-directive expression before it. E.g. it collects this `"use client"` while it shouldn’t:

```js
import 'foo'

"use client"
```